packages/config/config/devices/0x0312/mp21zd.json

```diff
@@ -0,0 +1,177 @@
+{
+	"manufacturer": "Minoston",
+	"manufacturerId": "0x0312",
+	"label": "MP21ZD",
+	"description": "Smart Plug Dimmer",
+	"devices": [
+		{
+			"productType": "0xff00",
+			"productId": "0xff0d"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"2": {
+			"label": "Status LED Configuration",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED on when load off, LED off when load on",
+					"value": 0
+				},
+				{
+					"label": "LED on when load on, LED off when load off",
+					"value": 1
+				},
+				{
+					"label": "LED always off",
+					"value": 2
+				},
+				{
+					"label": "LED always on",
+					"value": 3
+				}
+			]
+		},
+		"4": {
+			"label": "Auto-Off Timer",
+			"valueSize": 4,
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"6": {
+			"label": "Auto-On Timer",
+			"valueSize": 4,
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"7": {
+			"label": "Night Light Brightness",
+			"valueSize": 1,
+			"unit": "10%",
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 2,
+			"unsigned": true
+		},
+		"8": {
+			"label": "State After Power Failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				},
+				{
+					"label": "Last state",
+					"value": 2
+				}
+			]
+		},
+		"9": {
+			"label": "Dimmer Ramp Rate (On/Off)",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 1,
+			"unsigned": true
+		},
+		"10": {
+			"label": "Dimmer Ramp Rate (Change Brightness)",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 4,
+			"unsigned": true
+		},
+		"11": {
+			"label": "Minimum Brightness",
+			"valueSize": 1,
+			"unit": "%",
```

Comment:
```suggestion
```
Nitpicking,  but brightness in Z-Wave is not in %. See the note above https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=readwrite-only